tests/integration/events.js

```diff
@@ -388,6 +390,28 @@ test('store.on("change") with adding one', function (t) {
   })
 })
 
+test('store.on("add") should emit after store.add() promise resolved', function (t) {
```

Comment:
I think this should say: 
store.on("add") should _not_ emit after store.add() promise resolved